src/PinAssignment/PackagePinsModel.h

```diff
@@ -19,12 +19,48 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 #pragma once
+#include <QObject>
+#include <QStringList>
+#include <QVector>
 
 namespace FOEDAG {
 
-class PackagePinsModel {
+enum PinData {
+  PinName = 0,
```

Comment:
Temp to test ?

Reply:
didn't understand

Reply:
Get definitions for columns from hardware team.

Reply:
We will come back to this in future PR.